### PR TITLE
French Translation 

### PR DESCRIPTION
**Pull request by warquys**
I translated the Readme into French, for French people who are not competent enough to read the English even with google translator (they exist).

### CODE REVIEW

File: Localization/README-ES.md [hunk 1]
```diff
@@ -18,8 +18,10 @@ Todos los eventos de EXILED están hechos con Harmony, lo que significa que no e
  - Segundo, ya que todo el código relacionado con EXILED no está integrado en el código del servidor, cosas como actualizaciones pequeñas tendrán un pequeño (si acaso) efecto. Esto permite que siga siendo compatible con actualizaciones futuras, también agiliza el proceso de actualización cuando de verdad *es* necesario.
 
 # READMEs Traducidas
+- [Русский](https://github.com/Exiled-Team/EXILED/blob/master/Localization/README-Русский.md)
 - [中文](https://github.com/Exiled-Team/EXILED/blob/master/Localization/README-中文.md)
-- [English](https://github.com/Exiled-Team/EXILED/blob/master/README.md)
```
Comment (moddedmcplayer) [inline Localization/README-ES.md:22]: ummm

**Review by moddedmcplayer — CHANGES_REQUESTED** [review]

File: Localization/README-ES.md (hunk 1 above)
Comment (warquys) [reply Localization/README-ES.md:22]: yea sory

**Review by moddedmcplayer — DISMISSED** [review]

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]
Do not target dev branch.

**Comment by warquys** [comment]
I modified the development branch so I can't push to another branch without pushing the whole Dev branch

**Comment by warquys** [comment]
https://github.com/Exiled-Team/EXILED/pull/1443

**Pull request closed without merging**